mobile/src/bundles/common/helpers/get-years-title/get-years-title.ts

```diff
@@ -0,0 +1,6 @@
+const getYearsTitle = (value?: number): string => {
+    const singleYear = 1;
+    return value ? `${value} year${value === singleYear ? '' : 's'}` : 'no';
```

Comment:
Conditional operator in Conditional operator. Try to write few unit test for it)

Reply:
we can simplify it